### ISSUE TITLE
error in line 783

### ISSUE THREAD

**Issue by kikkauz** [kdoyen/openid-connect-php#4]
Shouldn't it be 
$public_key_xml = sprintf($public_key_xml, Utils::b64url2b64($key->n), Utils::b64url2b64($key->e)); 
?

**Comment by kdoyen** [comment]
You are correct, I will make the change, test, and update the test cases to cover this issue.

**Comment by dominik-riha** [comment]
@kdoyen any progress on this?
